qa/pytest_komodo/basic/test_rawtransactions.py

```diff
@@ -32,11 +32,13 @@ def test_rawtransactions(self, test_params):  # create, fund, sign, send calls
         txid = res[0].get('txid')
         vout = res[0].get('vout')
         base_amount = res[0].get('amount')
+        # python float() is double precision floating point number,
+        # createrawtransaction method expects C float (8 digits) value
+        # "{0:.8f}".format(value)) returns number string with 8 digit precision and float() corrects the type
         if isinstance(base_amount, Decimal):
-            amount = float(base_amount) * 0.9
-            print(amount)
+            amount = float("{0:.8f}".format(float(base_amount) * 0.9))
```

Comment:
Please use `decimal` for arithmetic operations. (https://docs.python.org/2/library/decimal.html)

Because https://docs.python.org/3/tutorial/floatingpoint.html

Reply:
Ok. Will need some time to change tests to use Decimal.
It probably will be better to submit such update in a different PR.